{"name": "test_broken", "status": "broken", "statusDetails": {"message": "Exception: oops", "trace": "def test_broken():\n>       raise Exception('oops')\nE       Exception: oops\n\ntest_report_allure_demo1.py:22: Exception"}, "start": 1613568330171, "stop": 1613568330171, "uuid": "ef8105f3-0d1e-4a73-a70b-12655609bcc7", "historyId": "4955e480929ffcf89f884f590f96942f", "testCaseId": "a920edf9a96b7ede79ed5579a7fe4ea7", "fullName": "test_report_allure_demo1#test_broken", "labels": [{"name": "suite", "value": "test_report_allure_demo1"}, {"name": "host", "value": "DESKTOP-39CMD4K"}, {"name": "thread", "value": "32908-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_report_allure_demo1"}]}